beacon-chain/powchain/service.go

```diff
@@ -613,9 +606,13 @@ func (s *Service) run(done <-chan struct{}) {
 
 	s.initPOWService()
 
-	ticker := time.NewTicker(1 * time.Second)
+	ticker := time.NewTicker(5 * time.Minute)
```

Comment:
can we instead change this to eth1 block time ?

Reply:
In that case, we don't need a second ticker. I've just combined this with the headTicker case in the select statement.